### PR TITLE
fix(helm): read values.yaml instead of values.toml

### LINKED ISSUE #822: helm install: Not reading `values.yaml` out of chart
As reported by @kmala, currently doing a `helm install` does not read the `values.yaml` file.

### PR DESCRIPTION
**Pull request by technosophos**
Closes #822

### CODE REVIEW

**Comment by bgrant0607** [comment]
@technosophos Please remove the dependency on the toml library, also.

**Comment by technosophos** [comment]
There is still a bit of code that uses the TOML library. I'm waiting for one last PR before I can pull that code out. Then I'll remove the entire `pkg/chart` package.

File: pkg/chart/chart.go [hunk 1]
```diff
@@ -93,9 +95,13 @@ func (c *Chart) ChartsDir() string {
 	return filepath.Join(c.loader.dir(), preCharts)
 }
 
-// LoadValues loads the contents of values.toml into a map
+// LoadValues loads the contents of values.tomlinto a map
```
Comment (fibonacci1729) [inline pkg/chart/chart.go]: nit: space

Comment (technosophos) [inline pkg/chart/chart.go]: Fixed. Will merge when tests pass. Thx.

**Pull request merged**